runtime/src/main/java/org/corfudb/runtime/collections/streaming/StreamPollingScheduler.java

```diff
@@ -139,8 +136,33 @@ private void tick() {
         scheduler.submit(new Tick());
     }
 
+    // This method is used for LR use case, where streamListeners listen to both the client stream
+    // and lrStatus table
     public void addTask(@Nonnull StreamListener streamListener, @Nonnull String namespace,
-                        @Nonnull String streamTag, @Nonnull List<String> tablesOfInterest,
+                        @Nonnull String streamTag, String corfuSystemNs, String corfuLrStatusStreamTag,
```

Comment:
we can get the corfuSystem namespace from the map instead of passing it as a separate argument

Reply:
also, can we make streamTag a list so that individual tags need not be passed separately?

Reply:
Sure, Can do that. But I am thinking of having it a map of ns->tags, just to keep this and the below layers transparent to the corfu stream tag. In case we want to support more system tags than just lrStatus, we won't need to change from this layer onwards.

Please let me know if you think otherwise!